client/src/components/colors.js

```diff
@@ -5,3 +5,10 @@ export const responseColors = [
   "#2db300",
   "#43A089"
 ];
+
+export const gaugeColors = ["#43A089", "#b5b5b5"];
+
+//Returns respective color from Response colors based on integer provided, int must be between 0 - 5
+export const getResponseColor = int => {
+  return responseColors[Math.floor(int === 5 ? 4.9 : int)];
+};
```

Comment:
This is something that's good to put tests around - then you don't need the comment because the functionality is documented by the tests